testsuite/sequential_execution_performance.py

```diff
@@ -4,7 +4,7 @@
 
 # Set the tps threshold for block size 1k, 10k and 50k
 BLOCK_SIZES = ["1k", "10k", "50k"]
-THRESHOLDS = {"1k": 3200, "10k": 4300, "50k": 7400}
+THRESHOLDS = {"1k": 3500, "10k": 5200, "50k": 12600}
```

Comment:
Please note that the squential test is significantly faster after I removed the sharded block-stm execution from its path. That is why we need to tune these thresholds as well as parallel execution speed up thresholds.